src/Popover/Popover.js

```diff
@@ -90,22 +91,24 @@ Popover.propTypes = {
     disableEdgeDetection: PropTypes.bool,
     disableStyles: PropTypes.bool,
     noArrow: PropTypes.bool,
+    parentElement: PropTypes.shape({ current: PropTypes.any }),
     placement: PropTypes.oneOf(POPPER_PLACEMENTS),
     popperProps: PropTypes.object,
     onClickOutside: PropTypes.func,
     onEscapeKey: PropTypes.func
 };
 
 Popover.defaultProps = {
-    onClickOutside: () => {},
-    onEscapeKey: () => {}
+    onClickOutside: () => { },
+    onEscapeKey: () => { }
 };
 
 Popover.propDescriptions = {
     body: 'Node(s) to render in the overlay.',
     control: 'Node to render as the reference element (that the `body` will be placed in relation to).',
     disableEdgeDetection: 'Set to **true** to render popover without edge detection so popover will not flip from top to bottom with scroll.',
     noArrow: 'Set to **true** to render a popover without an arrow.',
+    parentElement: 'A reference (useRef) to an element of which the Popover will be positioned and sized relatively. Leave it empty to let it be done automatically.',
```

Comment:
Can we call this `parentRef` to be even more explicit?

And if we're going to mention `useRef()`, can we clarify the function itself shouldn't be passed in, but rather the return object from calling it?

Reply:
Also, the description seems to imply ("let it be done") that positioning and sizing **relative to its parent** _can_ happen automatically if left empty. Is that the case?

Reply:
Sadly, it's not. If it was, this PR wouldn't need to exist at all ;)
I had no idea how to let the developer know he should put useRef's _result_ there... Your idea of `parentRef` seems reasonable.

I tried to put some comments into the code displayed in the documentation website but it's impossible :(